mrp_multi_level/views/product_mrp_area_views.xml

```diff
@@ -53,7 +53,17 @@
                             <field name="mrp_transit_delay" invisible="1"/>
                             <field name="mrp_inspection_delay" invisible="1"/>
                             <field name="mrp_minimum_stock"/>
-                            <field name="mrp_minimum_order_qty"/>
+                            <label for="mrp_minimum_order_qty"/>
+                            <div name="mrp_minimum_order_qty" class="o_row">
+                                <field name="mrp_minimum_order_qty"/>
+                                <span name="update_min_qty">
+                                    <button string="Update Min Qty" type="object"
```

Comment:
I would prefer a main string like "Get from main supplier" (minimum qty can be implicit as it is just as its side in the same line)